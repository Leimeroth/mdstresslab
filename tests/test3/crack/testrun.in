% Read in atomic configuration and species information
read
    spec,species
    conf,config
end

% Set up the grid for computing the stress field
grid
    gfile,grid
end

% Define the KIM model used to compute the atomic interactions
potential
    modl,SW_BalamaneHauchShi_2017Brittle_Si__MO_381114941873_002
end

% Specify whether to decompose stress into unique and non-unique parts
uniqueness
    project,F
end

% Setup and begin stress calculation
stress
hardy,pk1,slmo,20
end

stop
