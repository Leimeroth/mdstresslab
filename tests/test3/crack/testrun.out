   
   ----------------------------------------------------------------------------
   
   M D S t r e s s L a b
   
   
   This code is distributed under the CDDL open source license
   
   Authors: N. C. Admal, Min Shi and E. B. Tadmor (University of Minnesota)
   
   Version 2.0.0 MPI
   
   For more information contact tadmor@umn.edu
   
   ----------------------------------------------------------------------------
   
   Number of processor(s) used: 1

 **Reading species data**
   Species on file:
     Si          0.0028085000000000
   Total number of species in file = 1

 **Reading configuration data**
   Dimension        = 3
   Number of atoms  = 214240
   Reference sample size     =    20007.9225516967198928    3841.0296255997400294      27.1547499999999999
   Deformed sample size      =    20007.9272606268532400    3879.4322604266421877      27.1547499999999999
   PBC                       =               F              F              T
   Total number of species in the configuration file = 1
   Species list in configuration:
   Si

 **Reading grid data**
   Read grid from file first PK               = grid.data

 **Reading interatomic potential**
   Model name is: SW_BalamaneHauchShi_2017Brittle_Si__MO_381114941873_002

 **Reading rigidity settings**
   Projection =          F

 **Reading stress settings**
   Calculating Hardy stress:     T
   Calculating Hardy stress on reference configuration (First PK/engineering stress):     T
   First PK-Hardy average domain: sphere_linear_mollifying
   Using shperical averaging domain on reference configuration:     T
   Only calculating the first PK stress.
   Using MLS to push the first PK stress to Cauchy stress:     F
   Calculating pushed cauchy stress:     F

 **Start calculating the stress tensor**
   Starting force calculation :
   Cutoff radius from model =        3.7711800000000002
   Turning off pbc
   Total number of atoms = 267748
   number of model routine names = 8
   Constructing atom neighbor list
   Building bins in this region MAX:    10197.7299918584758416    2129.8315362033686142      29.9233762665179768
   Building bins in this region MIN:     9795.1684015909504524    1712.3612526037220505      -2.8096150030748905
   nxbin: 53; dxbin:        7.5955017031608563
   nybin: 55; dybin:        7.5903687927208470
   nzbin: 4; dzbin:        8.1832478173982164
   nbin: 11660
   max_pairs_per_atom: 86
   Allocating array size of total_neigh, which is the total # of neighbors =   17756987
   Neighbor List Construction Finish!

 **Calculating the first pk stress tensor**
   
   Looping over all bonds to calculate the first pk stress tensors.

 **Setting the first PK stress grid**

 **Reading grid data from file**
   Reading grid points from file grid.data
   Number of grid points = 90601
   Grid points set finished!
   Constructing grid neighbor list
   Building bins in this region MAX:    10199.7347386452729552    2119.8243884534654171      29.8702250000999996
   Building bins in this region MIN:     9800.3475350234239158    1722.3573156182569619     -27.1547500000999982
   nxbin: 8; dxbin:       49.9234004527311299
   nybin: 8; dybin:       49.6833841044010569
   nzbin: 1; dzbin:       57.0249750002000013
   nbin: 64
   Performance alert: Maximum # of Neighbors per grid increased to 800
   Performance alert: Maximum # of Neighbors per grid increased to 1600
   Performance alert: Maximum # of Neighbors per grid increased to 3200
   Performance alert: Maximum # of Neighbors per grid increased to 6400
   Performance alert: Maximum # of Neighbors per grid increased to 12800
   Performance alert: Maximum # of Neighbors per grid increased to 25600
   Grid Neighbor List Construction Finish!
   Grid Neighbor List Construction cost:      468.7792239999999993
   Dividing grid points. 1 processor(s) has(have) 90601 grid points.
   Dividing grid points. 0 processor(s) has(have) 90600 grid points.
   
   Looping over all bonds to calculate the first pk-Hardy stress tensors.
   Grid point first pk-Hardy: 100
   Grid point first pk-Hardy: 200
   Grid point first pk-Hardy: 300
   Grid point first pk-Hardy: 400
   Grid point first pk-Hardy: 500
   Grid point first pk-Hardy: 600
   Grid point first pk-Hardy: 700
   Grid point first pk-Hardy: 800
   Grid point first pk-Hardy: 900
   Grid point first pk-Hardy: 1000
   Grid point first pk-Hardy: 1100
   Grid point first pk-Hardy: 1200
   Grid point first pk-Hardy: 1300
   Grid point first pk-Hardy: 1400
   Grid point first pk-Hardy: 1500
   Grid point first pk-Hardy: 1600
   Grid point first pk-Hardy: 1700
   Grid point first pk-Hardy: 1800
   Grid point first pk-Hardy: 1900
   Grid point first pk-Hardy: 2000
   Grid point first pk-Hardy: 2100
   Grid point first pk-Hardy: 2200
   Grid point first pk-Hardy: 2300
   Grid point first pk-Hardy: 2400
   Grid point first pk-Hardy: 2500
   Grid point first pk-Hardy: 2600
   Grid point first pk-Hardy: 2700
   Grid point first pk-Hardy: 2800
   Grid point first pk-Hardy: 2900
   Grid point first pk-Hardy: 3000
   Grid point first pk-Hardy: 3100
   Grid point first pk-Hardy: 3200
   Grid point first pk-Hardy: 3300
   Grid point first pk-Hardy: 3400
   Grid point first pk-Hardy: 3500
   Grid point first pk-Hardy: 3600
   Grid point first pk-Hardy: 3700
   Grid point first pk-Hardy: 3800
   Grid point first pk-Hardy: 3900
   Grid point first pk-Hardy: 4000
   Grid point first pk-Hardy: 4100
   Grid point first pk-Hardy: 4200
   Grid point first pk-Hardy: 4300
   Grid point first pk-Hardy: 4400
   Grid point first pk-Hardy: 4500
   Grid point first pk-Hardy: 4600
   Grid point first pk-Hardy: 4700
   Grid point first pk-Hardy: 4800
   Grid point first pk-Hardy: 4900
   Grid point first pk-Hardy: 5000
   Grid point first pk-Hardy: 5100
   Grid point first pk-Hardy: 5200
   Grid point first pk-Hardy: 5300
   Grid point first pk-Hardy: 5400
   Grid point first pk-Hardy: 5500
   Grid point first pk-Hardy: 5600
   Grid point first pk-Hardy: 5700
   Grid point first pk-Hardy: 5800
   Grid point first pk-Hardy: 5900
   Grid point first pk-Hardy: 6000
   Grid point first pk-Hardy: 6100
   Grid point first pk-Hardy: 6200
   Grid point first pk-Hardy: 6300
   Grid point first pk-Hardy: 6400
   Grid point first pk-Hardy: 6500
   Grid point first pk-Hardy: 6600
   Grid point first pk-Hardy: 6700
   Grid point first pk-Hardy: 6800
   Grid point first pk-Hardy: 6900
   Grid point first pk-Hardy: 7000
   Grid point first pk-Hardy: 7100
   Grid point first pk-Hardy: 7200
   Grid point first pk-Hardy: 7300
   Grid point first pk-Hardy: 7400
   Grid point first pk-Hardy: 7500
   Grid point first pk-Hardy: 7600
   Grid point first pk-Hardy: 7700
   Grid point first pk-Hardy: 7800
   Grid point first pk-Hardy: 7900
   Grid point first pk-Hardy: 8000
   Grid point first pk-Hardy: 8100
   Grid point first pk-Hardy: 8200
   Grid point first pk-Hardy: 8300
   Grid point first pk-Hardy: 8400
   Grid point first pk-Hardy: 8500
   Grid point first pk-Hardy: 8600
   Grid point first pk-Hardy: 8700
   Grid point first pk-Hardy: 8800
   Grid point first pk-Hardy: 8900
   Grid point first pk-Hardy: 9000
   Grid point first pk-Hardy: 9100
   Grid point first pk-Hardy: 9200
   Grid point first pk-Hardy: 9300
   Grid point first pk-Hardy: 9400
   Grid point first pk-Hardy: 9500
   Grid point first pk-Hardy: 9600
   Grid point first pk-Hardy: 9700
   Grid point first pk-Hardy: 9800
   Grid point first pk-Hardy: 9900
   Grid point first pk-Hardy: 10000
   Grid point first pk-Hardy: 10100
   Grid point first pk-Hardy: 10200
   Grid point first pk-Hardy: 10300
   Grid point first pk-Hardy: 10400
   Grid point first pk-Hardy: 10500
   Grid point first pk-Hardy: 10600
   Grid point first pk-Hardy: 10700
   Grid point first pk-Hardy: 10800
   Grid point first pk-Hardy: 10900
   Grid point first pk-Hardy: 11000
   Grid point first pk-Hardy: 11100
   Grid point first pk-Hardy: 11200
   Grid point first pk-Hardy: 11300
   Grid point first pk-Hardy: 11400
   Grid point first pk-Hardy: 11500
   Grid point first pk-Hardy: 11600
   Grid point first pk-Hardy: 11700
   Grid point first pk-Hardy: 11800
   Grid point first pk-Hardy: 11900
   Grid point first pk-Hardy: 12000
   Grid point first pk-Hardy: 12100
   Grid point first pk-Hardy: 12200
   Grid point first pk-Hardy: 12300
   Grid point first pk-Hardy: 12400
   Grid point first pk-Hardy: 12500
   Grid point first pk-Hardy: 12600
   Grid point first pk-Hardy: 12700
   Grid point first pk-Hardy: 12800
   Grid point first pk-Hardy: 12900
   Grid point first pk-Hardy: 13000
   Grid point first pk-Hardy: 13100
   Grid point first pk-Hardy: 13200
   Grid point first pk-Hardy: 13300
   Grid point first pk-Hardy: 13400
   Grid point first pk-Hardy: 13500
   Grid point first pk-Hardy: 13600
   Grid point first pk-Hardy: 13700
   Grid point first pk-Hardy: 13800
   Grid point first pk-Hardy: 13900
   Grid point first pk-Hardy: 14000
   Grid point first pk-Hardy: 14100
   Grid point first pk-Hardy: 14200
   Grid point first pk-Hardy: 14300
   Grid point first pk-Hardy: 14400
   Grid point first pk-Hardy: 14500
   Grid point first pk-Hardy: 14600
   Grid point first pk-Hardy: 14700
   Grid point first pk-Hardy: 14800
   Grid point first pk-Hardy: 14900
   Grid point first pk-Hardy: 15000
   Grid point first pk-Hardy: 15100
   Grid point first pk-Hardy: 15200
   Grid point first pk-Hardy: 15300
   Grid point first pk-Hardy: 15400
   Grid point first pk-Hardy: 15500
   Grid point first pk-Hardy: 15600
   Grid point first pk-Hardy: 15700
   Grid point first pk-Hardy: 15800
   Grid point first pk-Hardy: 15900
   Grid point first pk-Hardy: 16000
   Grid point first pk-Hardy: 16100
   Grid point first pk-Hardy: 16200
   Grid point first pk-Hardy: 16300
   Grid point first pk-Hardy: 16400
   Grid point first pk-Hardy: 16500
   Grid point first pk-Hardy: 16600
   Grid point first pk-Hardy: 16700
   Grid point first pk-Hardy: 16800
   Grid point first pk-Hardy: 16900
   Grid point first pk-Hardy: 17000
   Grid point first pk-Hardy: 17100
   Grid point first pk-Hardy: 17200
   Grid point first pk-Hardy: 17300
   Grid point first pk-Hardy: 17400
   Grid point first pk-Hardy: 17500
   Grid point first pk-Hardy: 17600
   Grid point first pk-Hardy: 17700
   Grid point first pk-Hardy: 17800
   Grid point first pk-Hardy: 17900
   Grid point first pk-Hardy: 18000
   Grid point first pk-Hardy: 18100
   Grid point first pk-Hardy: 18200
   Grid point first pk-Hardy: 18300
   Grid point first pk-Hardy: 18400
   Grid point first pk-Hardy: 18500
   Grid point first pk-Hardy: 18600
   Grid point first pk-Hardy: 18700
   Grid point first pk-Hardy: 18800
   Grid point first pk-Hardy: 18900
   Grid point first pk-Hardy: 19000
   Grid point first pk-Hardy: 19100
   Grid point first pk-Hardy: 19200
   Grid point first pk-Hardy: 19300
   Grid point first pk-Hardy: 19400
   Grid point first pk-Hardy: 19500
   Grid point first pk-Hardy: 19600
   Grid point first pk-Hardy: 19700
   Grid point first pk-Hardy: 19800
   Grid point first pk-Hardy: 19900
   Grid point first pk-Hardy: 20000
   Grid point first pk-Hardy: 20100
   Grid point first pk-Hardy: 20200
   Grid point first pk-Hardy: 20300
   Grid point first pk-Hardy: 20400
   Grid point first pk-Hardy: 20500
   Grid point first pk-Hardy: 20600
   Grid point first pk-Hardy: 20700
   Grid point first pk-Hardy: 20800
   Grid point first pk-Hardy: 20900
   Grid point first pk-Hardy: 21000
   Grid point first pk-Hardy: 21100
   Grid point first pk-Hardy: 21200
   Grid point first pk-Hardy: 21300
   Grid point first pk-Hardy: 21400
   Grid point first pk-Hardy: 21500
   Grid point first pk-Hardy: 21600
   Grid point first pk-Hardy: 21700
   Grid point first pk-Hardy: 21800
   Grid point first pk-Hardy: 21900
   Grid point first pk-Hardy: 22000
   Grid point first pk-Hardy: 22100
   Grid point first pk-Hardy: 22200
   Grid point first pk-Hardy: 22300
   Grid point first pk-Hardy: 22400
   Grid point first pk-Hardy: 22500
   Grid point first pk-Hardy: 22600
   Grid point first pk-Hardy: 22700
   Grid point first pk-Hardy: 22800
   Grid point first pk-Hardy: 22900
   Grid point first pk-Hardy: 23000
   Grid point first pk-Hardy: 23100
   Grid point first pk-Hardy: 23200
   Grid point first pk-Hardy: 23300
   Grid point first pk-Hardy: 23400
   Grid point first pk-Hardy: 23500
   Grid point first pk-Hardy: 23600
   Grid point first pk-Hardy: 23700
   Grid point first pk-Hardy: 23800
   Grid point first pk-Hardy: 23900
   Grid point first pk-Hardy: 24000
   Grid point first pk-Hardy: 24100
   Grid point first pk-Hardy: 24200
   Grid point first pk-Hardy: 24300
   Grid point first pk-Hardy: 24400
   Grid point first pk-Hardy: 24500
   Grid point first pk-Hardy: 24600
   Grid point first pk-Hardy: 24700
   Grid point first pk-Hardy: 24800
   Grid point first pk-Hardy: 24900
   Grid point first pk-Hardy: 25000
   Grid point first pk-Hardy: 25100
   Grid point first pk-Hardy: 25200
   Grid point first pk-Hardy: 25300
   Grid point first pk-Hardy: 25400
   Grid point first pk-Hardy: 25500
   Grid point first pk-Hardy: 25600
   Grid point first pk-Hardy: 25700
   Grid point first pk-Hardy: 25800
   Grid point first pk-Hardy: 25900
   Grid point first pk-Hardy: 26000
   Grid point first pk-Hardy: 26100
   Grid point first pk-Hardy: 26200
   Grid point first pk-Hardy: 26300
   Grid point first pk-Hardy: 26400
   Grid point first pk-Hardy: 26500
   Grid point first pk-Hardy: 26600
   Grid point first pk-Hardy: 26700
   Grid point first pk-Hardy: 26800
   Grid point first pk-Hardy: 26900
   Grid point first pk-Hardy: 27000
   Grid point first pk-Hardy: 27100
   Grid point first pk-Hardy: 27200
   Grid point first pk-Hardy: 27300
   Grid point first pk-Hardy: 27400
   Grid point first pk-Hardy: 27500
   Grid point first pk-Hardy: 27600
   Grid point first pk-Hardy: 27700
   Grid point first pk-Hardy: 27800
   Grid point first pk-Hardy: 27900
   Grid point first pk-Hardy: 28000
   Grid point first pk-Hardy: 28100
   Grid point first pk-Hardy: 28200
   Grid point first pk-Hardy: 28300
   Grid point first pk-Hardy: 28400
   Grid point first pk-Hardy: 28500
   Grid point first pk-Hardy: 28600
   Grid point first pk-Hardy: 28700
   Grid point first pk-Hardy: 28800
   Grid point first pk-Hardy: 28900
   Grid point first pk-Hardy: 29000
   Grid point first pk-Hardy: 29100
   Grid point first pk-Hardy: 29200
   Grid point first pk-Hardy: 29300
   Grid point first pk-Hardy: 29400
   Grid point first pk-Hardy: 29500
   Grid point first pk-Hardy: 29600
   Grid point first pk-Hardy: 29700
   Grid point first pk-Hardy: 29800
   Grid point first pk-Hardy: 29900
   Grid point first pk-Hardy: 30000
   Grid point first pk-Hardy: 30100
   Grid point first pk-Hardy: 30200
   Grid point first pk-Hardy: 30300
   Grid point first pk-Hardy: 30400
   Grid point first pk-Hardy: 30500
   Grid point first pk-Hardy: 30600
   Grid point first pk-Hardy: 30700
   Grid point first pk-Hardy: 30800
   Grid point first pk-Hardy: 30900
   Grid point first pk-Hardy: 31000
   Grid point first pk-Hardy: 31100
   Grid point first pk-Hardy: 31200
   Grid point first pk-Hardy: 31300
   Grid point first pk-Hardy: 31400
   Grid point first pk-Hardy: 31500
   Grid point first pk-Hardy: 31600
   Grid point first pk-Hardy: 31700
   Grid point first pk-Hardy: 31800
   Grid point first pk-Hardy: 31900
   Grid point first pk-Hardy: 32000
   Grid point first pk-Hardy: 32100
   Grid point first pk-Hardy: 32200
   Grid point first pk-Hardy: 32300
   Grid point first pk-Hardy: 32400
   Grid point first pk-Hardy: 32500
   Grid point first pk-Hardy: 32600
   Grid point first pk-Hardy: 32700
   Grid point first pk-Hardy: 32800
   Grid point first pk-Hardy: 32900
   Grid point first pk-Hardy: 33000
   Grid point first pk-Hardy: 33100
   Grid point first pk-Hardy: 33200
   Grid point first pk-Hardy: 33300
   Grid point first pk-Hardy: 33400
   Grid point first pk-Hardy: 33500
   Grid point first pk-Hardy: 33600
   Grid point first pk-Hardy: 33700
   Grid point first pk-Hardy: 33800
   Grid point first pk-Hardy: 33900
   Grid point first pk-Hardy: 34000
   Grid point first pk-Hardy: 34100
   Grid point first pk-Hardy: 34200
   Grid point first pk-Hardy: 34300
   Grid point first pk-Hardy: 34400
   Grid point first pk-Hardy: 34500
   Grid point first pk-Hardy: 34600
   Grid point first pk-Hardy: 34700
   Grid point first pk-Hardy: 34800
   Grid point first pk-Hardy: 34900
   Grid point first pk-Hardy: 35000
   Grid point first pk-Hardy: 35100
   Grid point first pk-Hardy: 35200
   Grid point first pk-Hardy: 35300
   Grid point first pk-Hardy: 35400
   Grid point first pk-Hardy: 35500
   Grid point first pk-Hardy: 35600
   Grid point first pk-Hardy: 35700
   Grid point first pk-Hardy: 35800
   Grid point first pk-Hardy: 35900
   Grid point first pk-Hardy: 36000
   Grid point first pk-Hardy: 36100
   Grid point first pk-Hardy: 36200
   Grid point first pk-Hardy: 36300
   Grid point first pk-Hardy: 36400
   Grid point first pk-Hardy: 36500
   Grid point first pk-Hardy: 36600
   Grid point first pk-Hardy: 36700
   Grid point first pk-Hardy: 36800
   Grid point first pk-Hardy: 36900
   Grid point first pk-Hardy: 37000
   Grid point first pk-Hardy: 37100
   Grid point first pk-Hardy: 37200
   Grid point first pk-Hardy: 37300
   Grid point first pk-Hardy: 37400
   Grid point first pk-Hardy: 37500
   Grid point first pk-Hardy: 37600
   Grid point first pk-Hardy: 37700
   Grid point first pk-Hardy: 37800
   Grid point first pk-Hardy: 37900
   Grid point first pk-Hardy: 38000
   Grid point first pk-Hardy: 38100
   Grid point first pk-Hardy: 38200
   Grid point first pk-Hardy: 38300
   Grid point first pk-Hardy: 38400
   Grid point first pk-Hardy: 38500
   Grid point first pk-Hardy: 38600
   Grid point first pk-Hardy: 38700
   Grid point first pk-Hardy: 38800
   Grid point first pk-Hardy: 38900
   Grid point first pk-Hardy: 39000
   Grid point first pk-Hardy: 39100
   Grid point first pk-Hardy: 39200
   Grid point first pk-Hardy: 39300
   Grid point first pk-Hardy: 39400
   Grid point first pk-Hardy: 39500
   Grid point first pk-Hardy: 39600
   Grid point first pk-Hardy: 39700
   Grid point first pk-Hardy: 39800
   Grid point first pk-Hardy: 39900
   Grid point first pk-Hardy: 40000
   Grid point first pk-Hardy: 40100
   Grid point first pk-Hardy: 40200
   Grid point first pk-Hardy: 40300
   Grid point first pk-Hardy: 40400
   Grid point first pk-Hardy: 40500
   Grid point first pk-Hardy: 40600
   Grid point first pk-Hardy: 40700
   Grid point first pk-Hardy: 40800
   Grid point first pk-Hardy: 40900
   Grid point first pk-Hardy: 41000
   Grid point first pk-Hardy: 41100
   Grid point first pk-Hardy: 41200
   Grid point first pk-Hardy: 41300
   Grid point first pk-Hardy: 41400
   Grid point first pk-Hardy: 41500
   Grid point first pk-Hardy: 41600
   Grid point first pk-Hardy: 41700
   Grid point first pk-Hardy: 41800
   Grid point first pk-Hardy: 41900
   Grid point first pk-Hardy: 42000
   Grid point first pk-Hardy: 42100
   Grid point first pk-Hardy: 42200
   Grid point first pk-Hardy: 42300
   Grid point first pk-Hardy: 42400
   Grid point first pk-Hardy: 42500
   Grid point first pk-Hardy: 42600
   Grid point first pk-Hardy: 42700
   Grid point first pk-Hardy: 42800
   Grid point first pk-Hardy: 42900
   Grid point first pk-Hardy: 43000
   Grid point first pk-Hardy: 43100
   Grid point first pk-Hardy: 43200
   Grid point first pk-Hardy: 43300
   Grid point first pk-Hardy: 43400
   Grid point first pk-Hardy: 43500
   Grid point first pk-Hardy: 43600
   Grid point first pk-Hardy: 43700
   Grid point first pk-Hardy: 43800
   Grid point first pk-Hardy: 43900
   Grid point first pk-Hardy: 44000
   Grid point first pk-Hardy: 44100
   Grid point first pk-Hardy: 44200
   Grid point first pk-Hardy: 44300
   Grid point first pk-Hardy: 44400
   Grid point first pk-Hardy: 44500
   Grid point first pk-Hardy: 44600
   Grid point first pk-Hardy: 44700
   Grid point first pk-Hardy: 44800
   Grid point first pk-Hardy: 44900
   Grid point first pk-Hardy: 45000
   Grid point first pk-Hardy: 45100
   Grid point first pk-Hardy: 45200
   Grid point first pk-Hardy: 45300
   Grid point first pk-Hardy: 45400
   Grid point first pk-Hardy: 45500
   Grid point first pk-Hardy: 45600
   Grid point first pk-Hardy: 45700
   Grid point first pk-Hardy: 45800
   Grid point first pk-Hardy: 45900
   Grid point first pk-Hardy: 46000
   Grid point first pk-Hardy: 46100
   Grid point first pk-Hardy: 46200
   Grid point first pk-Hardy: 46300
   Grid point first pk-Hardy: 46400
   Grid point first pk-Hardy: 46500
   Grid point first pk-Hardy: 46600
   Grid point first pk-Hardy: 46700
   Grid point first pk-Hardy: 46800
   Grid point first pk-Hardy: 46900
   Grid point first pk-Hardy: 47000
   Grid point first pk-Hardy: 47100
   Grid point first pk-Hardy: 47200
   Grid point first pk-Hardy: 47300
   Grid point first pk-Hardy: 47400
   Grid point first pk-Hardy: 47500
   Grid point first pk-Hardy: 47600
   Grid point first pk-Hardy: 47700
   Grid point first pk-Hardy: 47800
   Grid point first pk-Hardy: 47900
   Grid point first pk-Hardy: 48000
   Grid point first pk-Hardy: 48100
   Grid point first pk-Hardy: 48200
   Grid point first pk-Hardy: 48300
   Grid point first pk-Hardy: 48400
   Grid point first pk-Hardy: 48500
   Grid point first pk-Hardy: 48600
   Grid point first pk-Hardy: 48700
   Grid point first pk-Hardy: 48800
   Grid point first pk-Hardy: 48900
   Grid point first pk-Hardy: 49000
   Grid point first pk-Hardy: 49100
   Grid point first pk-Hardy: 49200
   Grid point first pk-Hardy: 49300
   Grid point first pk-Hardy: 49400
   Grid point first pk-Hardy: 49500
   Grid point first pk-Hardy: 49600
   Grid point first pk-Hardy: 49700
   Grid point first pk-Hardy: 49800
   Grid point first pk-Hardy: 49900
   Grid point first pk-Hardy: 50000
   Grid point first pk-Hardy: 50100
   Grid point first pk-Hardy: 50200
   Grid point first pk-Hardy: 50300
   Grid point first pk-Hardy: 50400
   Grid point first pk-Hardy: 50500
   Grid point first pk-Hardy: 50600
   Grid point first pk-Hardy: 50700
   Grid point first pk-Hardy: 50800
   Grid point first pk-Hardy: 50900
   Grid point first pk-Hardy: 51000
   Grid point first pk-Hardy: 51100
   Grid point first pk-Hardy: 51200
   Grid point first pk-Hardy: 51300
   Grid point first pk-Hardy: 51400
   Grid point first pk-Hardy: 51500
   Grid point first pk-Hardy: 51600
   Grid point first pk-Hardy: 51700
   Grid point first pk-Hardy: 51800
   Grid point first pk-Hardy: 51900
   Grid point first pk-Hardy: 52000
   Grid point first pk-Hardy: 52100
   Grid point first pk-Hardy: 52200
   Grid point first pk-Hardy: 52300
   Grid point first pk-Hardy: 52400
   Grid point first pk-Hardy: 52500
   Grid point first pk-Hardy: 52600
   Grid point first pk-Hardy: 52700
   Grid point first pk-Hardy: 52800
   Grid point first pk-Hardy: 52900
   Grid point first pk-Hardy: 53000
   Grid point first pk-Hardy: 53100
   Grid point first pk-Hardy: 53200
   Grid point first pk-Hardy: 53300
   Grid point first pk-Hardy: 53400
   Grid point first pk-Hardy: 53500
   Grid point first pk-Hardy: 53600
   Grid point first pk-Hardy: 53700
   Grid point first pk-Hardy: 53800
   Grid point first pk-Hardy: 53900
   Grid point first pk-Hardy: 54000
   Grid point first pk-Hardy: 54100
   Grid point first pk-Hardy: 54200
   Grid point first pk-Hardy: 54300
   Grid point first pk-Hardy: 54400
   Grid point first pk-Hardy: 54500
   Grid point first pk-Hardy: 54600
   Grid point first pk-Hardy: 54700
   Grid point first pk-Hardy: 54800
   Grid point first pk-Hardy: 54900
   Grid point first pk-Hardy: 55000
   Grid point first pk-Hardy: 55100
   Grid point first pk-Hardy: 55200
   Grid point first pk-Hardy: 55300
   Grid point first pk-Hardy: 55400
   Grid point first pk-Hardy: 55500
   Grid point first pk-Hardy: 55600
   Grid point first pk-Hardy: 55700
   Grid point first pk-Hardy: 55800
   Grid point first pk-Hardy: 55900
   Grid point first pk-Hardy: 56000
   Grid point first pk-Hardy: 56100
   Grid point first pk-Hardy: 56200
   Grid point first pk-Hardy: 56300
   Grid point first pk-Hardy: 56400
   Grid point first pk-Hardy: 56500
   Grid point first pk-Hardy: 56600
   Grid point first pk-Hardy: 56700
   Grid point first pk-Hardy: 56800
   Grid point first pk-Hardy: 56900
   Grid point first pk-Hardy: 57000
   Grid point first pk-Hardy: 57100
   Grid point first pk-Hardy: 57200
   Grid point first pk-Hardy: 57300
   Grid point first pk-Hardy: 57400
   Grid point first pk-Hardy: 57500
   Grid point first pk-Hardy: 57600
   Grid point first pk-Hardy: 57700
   Grid point first pk-Hardy: 57800
   Grid point first pk-Hardy: 57900
   Grid point first pk-Hardy: 58000
   Grid point first pk-Hardy: 58100
   Grid point first pk-Hardy: 58200
   Grid point first pk-Hardy: 58300
   Grid point first pk-Hardy: 58400
   Grid point first pk-Hardy: 58500
   Grid point first pk-Hardy: 58600
   Grid point first pk-Hardy: 58700
   Grid point first pk-Hardy: 58800
   Grid point first pk-Hardy: 58900
   Grid point first pk-Hardy: 59000
   Grid point first pk-Hardy: 59100
   Grid point first pk-Hardy: 59200
   Grid point first pk-Hardy: 59300
   Grid point first pk-Hardy: 59400
   Grid point first pk-Hardy: 59500
   Grid point first pk-Hardy: 59600
   Grid point first pk-Hardy: 59700
   Grid point first pk-Hardy: 59800
   Grid point first pk-Hardy: 59900
   Grid point first pk-Hardy: 60000
   Grid point first pk-Hardy: 60100
   Grid point first pk-Hardy: 60200
   Grid point first pk-Hardy: 60300
   Grid point first pk-Hardy: 60400
   Grid point first pk-Hardy: 60500
   Grid point first pk-Hardy: 60600
   Grid point first pk-Hardy: 60700
   Grid point first pk-Hardy: 60800
   Grid point first pk-Hardy: 60900
   Grid point first pk-Hardy: 61000
   Grid point first pk-Hardy: 61100
   Grid point first pk-Hardy: 61200
   Grid point first pk-Hardy: 61300
   Grid point first pk-Hardy: 61400
   Grid point first pk-Hardy: 61500
   Grid point first pk-Hardy: 61600
   Grid point first pk-Hardy: 61700
   Grid point first pk-Hardy: 61800
   Grid point first pk-Hardy: 61900
   Grid point first pk-Hardy: 62000
   Grid point first pk-Hardy: 62100
   Grid point first pk-Hardy: 62200
   Grid point first pk-Hardy: 62300
   Grid point first pk-Hardy: 62400
   Grid point first pk-Hardy: 62500
   Grid point first pk-Hardy: 62600
   Grid point first pk-Hardy: 62700
   Grid point first pk-Hardy: 62800
   Grid point first pk-Hardy: 62900
   Grid point first pk-Hardy: 63000
   Grid point first pk-Hardy: 63100
   Grid point first pk-Hardy: 63200
   Grid point first pk-Hardy: 63300
   Grid point first pk-Hardy: 63400
   Grid point first pk-Hardy: 63500
   Grid point first pk-Hardy: 63600
   Grid point first pk-Hardy: 63700
   Grid point first pk-Hardy: 63800
   Grid point first pk-Hardy: 63900
   Grid point first pk-Hardy: 64000
   Grid point first pk-Hardy: 64100
   Grid point first pk-Hardy: 64200
   Grid point first pk-Hardy: 64300
   Grid point first pk-Hardy: 64400
   Grid point first pk-Hardy: 64500
   Grid point first pk-Hardy: 64600
   Grid point first pk-Hardy: 64700
   Grid point first pk-Hardy: 64800
   Grid point first pk-Hardy: 64900
   Grid point first pk-Hardy: 65000
   Grid point first pk-Hardy: 65100
   Grid point first pk-Hardy: 65200
   Grid point first pk-Hardy: 65300
   Grid point first pk-Hardy: 65400
   Grid point first pk-Hardy: 65500
   Grid point first pk-Hardy: 65600
   Grid point first pk-Hardy: 65700
   Grid point first pk-Hardy: 65800
   Grid point first pk-Hardy: 65900
   Grid point first pk-Hardy: 66000
   Grid point first pk-Hardy: 66100
   Grid point first pk-Hardy: 66200
   Grid point first pk-Hardy: 66300
   Grid point first pk-Hardy: 66400
   Grid point first pk-Hardy: 66500
   Grid point first pk-Hardy: 66600
   Grid point first pk-Hardy: 66700
   Grid point first pk-Hardy: 66800
   Grid point first pk-Hardy: 66900
   Grid point first pk-Hardy: 67000
   Grid point first pk-Hardy: 67100
   Grid point first pk-Hardy: 67200
   Grid point first pk-Hardy: 67300
   Grid point first pk-Hardy: 67400
   Grid point first pk-Hardy: 67500
   Grid point first pk-Hardy: 67600
   Grid point first pk-Hardy: 67700
   Grid point first pk-Hardy: 67800
   Grid point first pk-Hardy: 67900
   Grid point first pk-Hardy: 68000
   Grid point first pk-Hardy: 68100
   Grid point first pk-Hardy: 68200
   Grid point first pk-Hardy: 68300
   Grid point first pk-Hardy: 68400
   Grid point first pk-Hardy: 68500
   Grid point first pk-Hardy: 68600
   Grid point first pk-Hardy: 68700
   Grid point first pk-Hardy: 68800
   Grid point first pk-Hardy: 68900
   Grid point first pk-Hardy: 69000
   Grid point first pk-Hardy: 69100
   Grid point first pk-Hardy: 69200
   Grid point first pk-Hardy: 69300
   Grid point first pk-Hardy: 69400
   Grid point first pk-Hardy: 69500
   Grid point first pk-Hardy: 69600
   Grid point first pk-Hardy: 69700
   Grid point first pk-Hardy: 69800
   Grid point first pk-Hardy: 69900
